queries.sql

```diff
@@ -92,3 +92,42 @@ WHERE date_of_birth >= '1990-01-01' AND date_of_birth <= '2000-12-31'
 GROUP BY species;
 
 SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/229166316-d001f8cb-aa8b-4a75-800d-e26630082805.png)

**[OPTIONAL]**

- [ ] In this query 11 records are returned but in actual 10 records should be returned 😨 You query is perfect 👍 but i guess there is something you missed from the previous milestone which is causing this issue 🤔 if you remember there was a transaction in the previous milestone which was deleting animals born after `2022-01-01` kindly run that transaction to keep your data up to date with the current milestone 👍

Reply:
Thank you for the review, let me check and update the records

Reply:
Thank you @Yerimah